rotkehlchen/db/filtering.py

```diff
@@ -310,8 +322,17 @@ def make(
         if tx_hash is not None:  # tx_hash means single result so make it as single filter
             filters.append(DBETHTransactionHashFilter(and_op=False, tx_hash=tx_hash))
         else:
-            if addresses is not None:
-                filter_query.join_clause = DBETHTransactionAddressFilter(and_op=False, addresses=addresses)  # noqa: E501
+            if addresses or asset or protocols:
```

Comment:
Explicit truthy checks

```suggestion
            if addresses is not None or asset is not None or protocols is not None:
```